classes/local/envbarlib.php

```diff
@@ -719,4 +720,50 @@ public static function add_menuuser(): array {
         return $navitem;
     }
 
+    /**
+     * Updates the stored timestamp for the next expected refresh. Called when last refresh is upadted and when
+     * the config is changed.
+     *
+     * @return void
+     * @throws \dml_exception
+     */
+    public static function reset_next_refresh_timestamp() {
+        $config = get_config('local_envbar');
```

Comment:
when I installed this it was broken at the start, the nextrefresh wasn't working in the envbar and also the check was wrong. After I change nextrefresh to something else it worked so I think there is either some logic missing somewhere (preferred) or an upgrade step missing.

Reply:
Can you elaborate how you installed it?

Reply:
I already had it installed and upgraded it

Reply:
Fixed. It correctly handles changes to forced config as well now.